client/webserver/live_test.go

```diff
@@ -583,6 +584,12 @@ func (c *TCore) EstimateRegistrationTxFee(host string, certI interface{}, assetI
 	}
 	return txFee, nil
 }
+func (c *TCore) ValidateAddress(address string, assetID uint32) (bool, error) {
+	return c.validAddr, nil
```

Comment:
Can we make this more useful for UI testing by applying a simple length requirement
```Go
return len(address) > 10
```